api/docs/org.jboss.tools.rsp.schema/src/main/resources/schema/typescript/protocol.unified.d.ts

```diff
@@ -1,5 +1,5 @@
 /* tslint:disable */
-// Generated using typescript-generator version 2.2.413 on 2019-06-03 14:27:41.
+// Generated using typescript-generator version 2.2.413 on 2019-06-05 13:54:35.
```

Comment:
This change should be removed from the PR for better clarity

Reply:
see above (same here, should not be included)